site-src/concepts/versioning.md

```diff
@@ -67,74 +95,61 @@ gateway.networking.k8s.io/channel: standard|experimental
+    * Removal of an API resource following [Kubernetes deprecation
   policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/)
+* All Channels:
+    * Changes to recommended conditions or reasons in status
+    * Loosened validation (including making required fields optional)
+    * Changes to conformance tests to match spec updates
+    * Introduction of a new API version which may include renamed fields or
+      anything else that is valid in a [new Kubernetes API
+      version](https://kubernetes.io/docs/reference/using-api/#api-versioning)
 
 ### Major version (e.g. v0.x to v1.0)
 * There are no API compatibility guarantees when the major version changes.
 
 ## Graduation Criteria
+For a resource, field, or feature to graduate from Experimental to Standard, it
+must meet the following criteria:
 
-### Resources
-
-#### Alpha -> Beta
-A resource to graduate from alpha to beta must meet the following criteria:
-
-* Implemented by several implementations.
-* Conformance test framework is in place, with some coverage of basic
-  functionality.
-* Validation is well thought out.
-* Most of the API surface is being exercised by users.
-* Approval from subproject owners + KEP reviewers.
-
-#### Beta -> GA
-
-A resource to graduate from beta to GA must meet the following criteria:
-
-* Almost all of the fields and behavior have conformance test coverage.
+* Full conformance test coverage.
 * Multiple conformant implementations.
 * Widespread implementation and usage.
-* At least 6 months of soak time as a beta API.
+* At least 6 months of soak time as an alpha API.
+* No significant changes for at least 1 minor release and 3 months
 * Approval from subproject owners + KEP reviewers.
 
-### Fields
-
-#### Experimental -> Standard
-As described above, field level stability is layered on top of beta and GA
-resources (no fields in alpha resources can be considered standard). The
-requirements for a field to graduate from experimental to standard depend on the
-API version of the resource it is a part of. For a field to be considered
-standard, it needs to meet the same criteria of the resource it is contained in.
+## Supported Versions
+This project aims to provide support for a wide range of Kubernetes versions with
+consistent upgrade experiences across versions. To accomplish that, we commit to:
 
-If a resource has graduated to beta, an experimental field must meet all of the
-beta graduation criteria before graduating to standard. Similarly, if a resource
-has graduated to GA, a field must meet all of the beta and GA graduation
-criteria. There is one slight variation here, instead of 6 months of soak time
-as a beta API, a field graduating to standard requires 6 months of soak time as an
-experimental field.
+1. Support a minimum of the most recent 5 Kubernetes minor versions.
+2. Ensure that all standard channel changes between v1beta1 and v1 are fully
```

Comment:
*nit* capitalization consistency. I see Standard Channel, Standard channel, standard channel in different places.
```suggestion
2. Ensure that all Standard Channel changes between v1beta1 and v1 are fully
```

Reply:
Updated, thanks!